pkg/kubelet/eviction/helpers.go

```diff
@@ -1259,7 +1259,7 @@ func evictionMessage(resourceToReclaim v1.ResourceName, pod *v1.Pod, stats stats
 				switch resourceToReclaim {
 				case v1.ResourceEphemeralStorage:
 					if containerStats.Rootfs != nil && containerStats.Rootfs.UsedBytes != nil && containerStats.Logs != nil && containerStats.Logs.UsedBytes != nil {
```

Comment:
You should also check here if `podStats.EphemeralStorage.UsedBytes` is not nil etc.